axon-cosmosdb/src/test/java/org/axonframework/extensions/cosmosdb/eventsourcing/tokenstore/CosmosTokenStoreIntegrationTest.java

```diff
@@ -447,8 +448,14 @@ void shouldBeAbleToStealAfterTimeoutHasPassed() throws InterruptedException {
         testSubject.storeToken(testToken, PROCESSOR_ONE, TEST_SEGMENT);
         assertThrows(UnableToClaimTokenException.class,
                      () -> storeWithDecreasedTimeout.fetchToken(PROCESSOR_ONE, TEST_SEGMENT));
-        Thread.sleep(200L);
-        assertDoesNotThrow(() -> storeWithDecreasedTimeout.fetchToken(PROCESSOR_ONE, TEST_SEGMENT));
+        await()
```

Comment:
Is this fix per chance accidentally added to this branch, whereas it belongs to the branch for PR #1?